IDEAS/Examples/Tutorial/DetailedHouse/package.mo

```diff
@@ -0,0 +1,107 @@
+within IDEAS.Examples.Tutorial;
+package DetailedHouse "Package with example for how to built a simple house, using the detailed building envelope component
+  models, HVAC and other components"
```

Comment:
"Package with example for how to built an office building (using the detailed building envelope component models within IDEAS.Buildings) with occupants, a radiator heating system connected to a heat pump, and a ventilation system."